src/notification/notification.service.ts

```diff
@@ -32,31 +39,58 @@ export default class NotificationService {
         }
 
         const msg = `${senderUser.fullName} שיתף איתך קובץ.`;
-        const reciverUserT = this.extractUserT(reciverUser);
-
-        const hiResponse:
-            | IMessageReceiptAPI
-            | IMessageReceiptAPI[] = await driver.sendDirectToUser(
-            msg,
-            reciverUserT
-        );
+        const reciverUserTs = this.extractUserTs(reciverUser);
 
-        log(
-            Severity.INFO,
-            Array.isArray(hiResponse) ? hiResponse[0].msg : hiResponse.msg,
-            "sendSharedFileNotification",
-            undefined,
-            hiResponse
+        await Promise.all(
+            reciverUserTs.map((userT) =>
+                driver
+                    .sendDirectToUser(msg, userT)
+                    .then(
+                        (
+                            hiResponse:
+                                | IMessageReceiptAPI
+                                | IMessageReceiptAPI[]
+                        ) =>
+                            log(
```

Comment:
So it console.logs everything? Maybe add a debug_mode envar?

Reply:
You can't really debug it but if you do, you would like to see these logs